src/dot/commons/pose/head_pose.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import cv2
+import mediapipe as mp
+import numpy as np
+
+mp_face_mesh = mp.solutions.face_mesh
+face_mesh = mp_face_mesh.FaceMesh(
+    min_detection_confidence=0.5, min_tracking_confidence=0.5
+)
+mp_drawing = mp.solutions.drawing_utils
+
+
+def pose_estimation(
+    image: np.array, roll: int = 3, pitch: int = 3, yaw: int = 3
+) -> int:
+    """Given an image and desired roll, pitch and yaw angles, returns whether head pose estimation meets requirements.
+
+    Args:
+        image (np.array): Image to estimate head pose.
+        roll (int, optional): Rotation margin in X axis. Defaults to 3.
+        pitch (int, optional): Rotation margin in Y axis. Defaults to 3.
+        yaw (int, optional): Rotation margin in Z axis. Defaults to 3.
```

Comment:
you can remove information about type hints and defaults inside the docstring.

Reply:
https://github.com/sensity-ai/dot/pull/36/commits/51f5b41eb2c74db22099cf9f9ca76979978bf4bd